style.css

```diff
@@ -0,0 +1,219 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 1.6rem;
+    box-sizing: border-box
+}
+
+.wrapper {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1200px;
+}
+
+a:visited {
+    text-decoration: none;
+    color: black;
+}
+
+/*HEADER*/
+
+.header-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #000;
+}
+
+.header-logo {
+    width: 10rem;
+}
+
+.header-title {
+    padding-left: 2rem;
+    font-size: 3rem;
+    color: #fcf4a5;
+}
+
+/*NAVBAR*/
+
+.navbar-container {
+    padding: 0 30rem;
+    display: flex;
+    justify-content: space-around;
+    font-size: 1.8rem;
+    font-weight: bold;
+}
+
+.navbar-link {
+    margin: 1rem 0;
+    text-decoration: none;
+    color: inherit;
+    border-bottom: 0.2rem solid rgba(0, 0, 0, 0);
+    transition: border-bottom 0.5s;
+}
+
+.navbar-link:hover {
+    border-bottom: 0.2rem solid #000;
+}
+
+/*PERSONALS*/
+
+.personals-upper {
+    background-color: black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.personals-photo {
+    height: 20rem;
+    border-radius: 50%;
+    position: relative;
+    top: 4rem;
+}
+
+.personals-lower {
+    padding-top: 3rem;
+    background-color: #fbfbfb;
+    display: flex;
+}
+
+.qrcode {
+    padding-left: 4rem;
+    padding-top: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.qrcode-image {
+    height: 13rem;
+    margin-top: 1rem;
+}
+
+.qrcode-description {
+    margin-bottom: 2rem;
+    padding: 3px 6px;
+    font-size: 1.8rem;
+    font-weight: bold;
+    text-decoration: none;
+    color: inherit;
+    transition: color 0.3s;
+}
+
+.qrcode-description:hover {
+    color: #fcda05;
+}
+
+.personals-text {
+    width: 100%;
+    padding-right: 18rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.personals-text__name {
+    text-transform: uppercase;
+    font-size: 3.5rem;
+    letter-spacing: 0.3rem;
+}
+
+.personals-text__data {
+    margin-top: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    line-height: 2.5rem;
+}
+
+.personals-text__line {
+    font-size: 1.8rem;
+}
+
+span {
```

Comment:
Don't write CSS that generic. If you forget about it later and add a `<span>` element, this CSS style will apply to it